bin/src/cli/run.rs

```diff
@@ -46,8 +49,9 @@ pub async fn exec(command: WatchtowerCommand) -> anyhow::Result<()> {
         ethereum_rpc_url: eth_rpcs[0].clone(),
         beacon_rpc_url: command.beacon_rpc_url,
         blob_contract: command.blob_contract,
-        // We will set it later in the syncer.
-        start_block: 0,
+        // We will set these later in the syncer.
+        da_start_block: 0,
+        next_fuel_block: BlockHeight::new(0),
```

Comment:
Nit: It would be cleaner to separate the types to something like `PartialDownloaderConfig` for the config before these are set, and then implement a conversion method that takes these parameters to construct the full `DownloaderConfig`. This might be overkill though, so not a blocker.

Reply:
yup, can address in another pr

Reply:
https://github.com/FuelLabs/network-watchtower/issues/18